### PR TITLE
Fix pytest and coverage

### PR DESCRIPTION
**Pull request by rafrafek**
Hi!

I've added `tests/__init__.py` so now I can just run `pytest` and it works. I will comment other changes in code.

### CODE REVIEW

File: tests/test_timecode.py [hunk 1]
```diff
@@ -67,11 +67,11 @@ def test_instance_creation_11(self):
         tc = Timecode('24', start_timecode=None, frames=12000)
         self.assertIsInstance(tc, Timecode)
 
-    def test_instance_creation_12(self):
+    def test_instance_creation_12_a(self):
         tc = Timecode('23.976')
         self.assertIsInstance(tc, Timecode)
 
-    def test_instance_creation_12(self):
```
Comment (rafrafek) [inline tests/test_timecode.py:74]: Duplicate method name

File: tests/test_timecode.py [hunk 2]
```diff
@@ -147,11 +147,11 @@ def test_instance_creation_30(self):
         tc = Timecode('ms', 421729315)
         self.assertIsInstance(tc, Timecode)
 
-    def test_instance_creation_31(self):
+    def test_instance_creation_31_a(self):
         tc = Timecode('24000/1000', '00:00:00:00')
         self.assertIsInstance(tc, Timecode)
 
-    def test_instance_creation_31(self):
```
Comment (rafrafek) [inline tests/test_timecode.py:154]: Duplicate method name

File: tests/test_timecode.py [hunk 3]
```diff
@@ -1477,7 +1477,7 @@ def test_le_overload(self):
         self.assertTrue(tc5 >= tc4)
         self.assertTrue(tc5 > tc4)
 
-    def test_gt_overload(self):
```
Comment (rafrafek) [inline tests/test_timecode.py:1480]: Duplicate method name

File: tests/test_timecode.py [hunk 4]
```diff
@@ -1704,7 +1703,10 @@ def test_fraction_lib_from_python3_raises_import_error_for_python2(self):
         This is purely done for increasing the code coverage to 100% under
         Python 3
         """
-        import mock
```
Comment (rafrafek) [inline tests/test_timecode.py:1707]: Does not work on newer versions of Python

File: tests/test_timecode.py [hunk 5]
```diff
@@ -1551,7 +1551,6 @@ def test_bug_report_31_part1(self):
 
         with self.assertRaises(ValueError) as cm:
             offset = a - b
-            c = Timecode('25', timecode3) + offset
```
Comment (rafrafek) [inline tests/test_timecode.py:1554]: This line was not covered. Probably exception is raised one line before

**Comment by eoyilmaz** [comment]
Hey sorry for my late answer, I'll look in to that soon 👍

**Comment by eoyilmaz** [comment]
Thanks for the PR

**Comment by rafrafek** [comment]
You're welcome :)

**Pull request merged**